_projects/2024/100419840/100419840.Rmd

```diff
@@ -0,0 +1,598 @@
+
+```{r}
+mi_grafico_mal <- mi_grafico_mal + scale_y_continuous(
+  limits = c(-80, 30),  # Establecer el límite superior en +30%
+  breaks = seq(-70, 20, by = 10),  # Mostrar etiquetas de cada 10 unidades
+  labels = function(y) ifelse(y == -80, "", paste0(y, "%")),  # Ocultar etiqueta en -80
+  expand = expansion(mult = c(0, 0.05))
+)
+
+mi_grafico_mal
+```
+
+Afterwards, the original colors of the highlighted regions are manually coded. (PONER AQUÍ EL LINK DEL BUSCADOR DE COLORES)
+
+```{r}
+mi_grafico_mal <- mi_grafico_mal + scale_color_manual(values = c("Media Nacional" = "#5E5E5E", "Baleares" = "#ECDC49", 
+                                "Madrid" = "#2C7BCB", "Canarias" = "#175695", "Otros" = "#D9D9D9")) 
+
+mi_grafico_mal
+```
+
+Then, the title, subtitle and caption are introduced using *labs*.
+
+```{r}
+
+mi_grafico_mal <- mi_grafico_mal +
+  labs(
+    title = "En Baleares y Madrid el precio de la vivienda ya está por encima del peor momento de la burbuja", 
+    subtitle = "Evolución trimestral del valor tasado de la vivienda libre en cada CCAA respecto al trimestre donde el precio de la vivienda tocó techo en cada\ncomunidad",
+    caption = "Fuente: Ministerio de Transportes y Movilidad Sostenible"
+  )
+
+mi_grafico_mal
+```
+Finally, the theme is established to accurately replicate the graph. Here is necessary to eliminate the legend, both axis labels and the grid lines of the x-axis. Regarding the y-axis, the grid lines are styled according to the original in grey. 
+
+```{r}
+mi_grafico_mal <- mi_grafico_mal + theme(
+  legend.position = "none",               # Remove the legend
+  axis.title.y = element_blank(),         # Remove y-axis title
+  axis.title.x = element_blank(),         # Remove x-axis title
+  panel.grid.major.x = element_blank(),   # Remove major grid lines on x-axis
+  panel.grid.major.y = element_line(color = "grey80", size = 0.5), # Style major y-axis grid lines
+  panel.grid.minor = element_blank(),     # Remove minor grid lines
+  plot.margin = margin(5, 80, 5, 5),      # Set plot margins
+  plot.title = element_text(face = "bold"), # Make the plot title bold
+  panel.background  = element_rect(fill = "white", color = NA) # Set plot background to white
+)
+
+
+mi_grafico_mal
+```
+
+Finally, *coord_cartesian* is employed to eliminate the limits that do not allow to see the highlighted regions' labels (concretely, the key command is clip = "off").
+
+```{r, fig.height=8, fig.width=17}
+mi_grafico_mal <- mi_grafico_mal + coord_cartesian(xlim = c(as.Date("1995-03-01"), 
+                                                            as.Date("2024-06-30")), clip = "off", expand = 0)
+
+mi_grafico_mal
+```
+
+
+However, this final graph has an issue since Ceuta and Melilla are not represented in the original graph. Therefore, the data is filtered to eliminate this two regions and then the code is executed altogether to represent the graph correctly.
+
+```{r, fig.height=8, fig.width=17}
```

Comment:
Please add the option `preview = TRUE` so that this image is highlighted in the gallery.